### PR TITLE
add jetbrains k8s filtering for knative serving (#45)

### LINKED ISSUE #45: Make JetBrains Kubernetes plugin avoid reporting errors on Ultimate for our schemas

### PR DESCRIPTION
**Pull request by lstocchi**
this resolves #45 
it works with https://github.com/redhat-developer/intellij-common/pull/55

@evidolob to give you some context. This patch is to prevent this error from happening https://github.com/redhat-developer/intellij-tekton/issues/263

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ideaVersion = IC-2019.3
+ideaVersion = IU-2019.3
```
Comment (evidolob) [inline gradle.properties:1]: Is there any reason to run IJ Ultimate instead of community?

Comment (lstocchi) [reply gradle.properties:1]: The Jetbrains kubernetes plugin is not available for the community edition so if we don't change it here the mapping on the `build.gradle` fails as there is no-existing plugin for IC

```
def versionsMap = ['IU-2019.2':'192.5728.98','IU-2019.3':'193.5233.144','IU-2020.1':'201.6668.99', 'IU-2020.2':'202.6397.93', 'IU-2020.3':'203.5981.152']

'com.intellij.kubernetes:' + versionsMap[ideaVersion]
```

Comment (lstocchi) [reply gradle.properties:1]: https://plugins.jetbrains.com/plugin/10485-kubernetes/versions

File: build.gradle [hunk 2]
```diff
@@ -16,10 +16,12 @@ repositories {
     maven { url 'https://repository.jboss.org' }
 }
 
+def versionsMap = ['IU-2019.2':'192.5728.98','IU-2019.3':'193.5233.144','IU-2020.1':'201.6668.99', 'IU-2020.2':'202.6397.93', 'IU-2020.3':'203.5981.152']
```
Comment (jeffmaury) [inline build.gradle:19]: 2019.2 is not required here as we start from 2019.3

**Review by jeffmaury — CHANGES_REQUESTED** [review]

**Comment by evidolob** [comment]
Not sure that is related to this PR, but I cannot edit knative resource:
Screenshot 2021-03-30 at 15 21 57

File: .github/workflows/IJ.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IU-2019.3, IU-2020.1, IU-2020.2, IU-2020.3]
+        IJ: [IU-2019.3, IU-2020.1, IU-2020.2, IU-2020.3, IU-2021.1]
```
Comment (jeffmaury) [inline .github/workflows/IJ.yml:17]: 2021.1 is not yet out

**Review by jeffmaury — CHANGES_REQUESTED** [review]

**Comment by lstocchi** [comment]
> Not sure that is related to this PR, but I cannot edit knative resource:
> Screenshot 2021-03-30 at 15 21 57

I just merged the `update` feature (https://github.com/redhat-developer/intellij-knative/pull/27) and i didn't rebase this one so it is normal.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by lstocchi** [comment]
@evidolob rebased if you want to try

**Review by evidolob — APPROVED** [review]

**Pull request merged**